shop/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.3/apache-maven-3.9.3-bin.zip
```

Comment:
You got a requirement to delete some files related to maven. You got in the IDE so we do not need it here:
![image](https://github.com/VladMiron4/Spring-training/assets/138880600/24039187-5ff6-4c36-90cd-3b681977aae0)